_includes/join-us/join-us-contribute-to-strimzi-band.html

```diff
@@ -1,121 +1,129 @@
-        If you come to the community calls, you will meet maintainers and other community members.
-        You can learn about the current work or future plans.
-        But you can also raise questions, discuss issues, or just chat about specific Strimzi use-cases or anything Strimzi-related.
-        We keep things friendly. In all our communications, on whatever platform, we expect everyone to follow a <a href="https://github.com/strimzi/governance/blob/main/CODE_OF_CONDUCT.md">code of conduct</a>.</p>
+      <p>
+        We welcome everyone to contribute in a respectful and inclusive environment. 
+        See the <a href="https://github.com/cncf/foundation/blob/main/code-of-conduct.md" target="_blank" rel="noopener noreferrer">CNCF Community Code of Conduct</a>.
+      </p>          
 
-      <h3 id="ask-for-help-and-help-others">Ask for help and help others</h3>
+      <hr class="section-divider">
 
-      <p>If you're new to Strimzi, you'll probably have lots of questions. 
-        We hope that our <a href="https://strimzi.io/quickstarts/">quickstarts</a>, <a href="https://strimzi.io/documentation/">documentation</a>, and <a href="https://www.youtube.com/playlist?list=PLpI4X8PMthYeCSpy9a-mGtvDbMgqGNYNy">tutorials</a> on our YouTube channels help you to get started. 
-        But don’t be afraid to ask if you have any questions or issues. 
-        The <a href="https://slack.cncf.io/"><code>#strimzi</code> Slack channel</a>, <a href="https://lists.cncf.io/g/cncf-strimzi-users">user mailing list</a>, and <a href="https://github.com/strimzi/strimzi-kafka-operator/discussions">GitHub discussions</a> are the best places to ask questions or ask for help.</p>
-
-      <p>To get useful help fast, it is always good to cover the following points:</p>
+      <!-- Connect with the community -->
+      <h3 style="display: flex; align-items: center; gap: 0.5em;">
+        <img src="/assets/images/community/community_icon_chatroom.svg" alt="" aria-hidden="true" style="height: 1.4em;">
+        Connect
+      </h3>
+      
+      <div class="grid-container">
+
+        <a class="tile" href="https://slack.cncf.io/" target="_blank" rel="noopener noreferrer">
+          <h6>Connect on Slack</h6>
+          <p>Connect on the <b>#strimzi</b> CNCF Slack channel</p>
+          <i class="fas fa-arrow-right arrow"></i>
+        </a>
+
+        <a class="tile" href="https://github.com/strimzi/strimzi-kafka-operator#strimzi-community-meetings" target="_blank" rel="noopener noreferrer">
+          <h6>Join community calls</h6>
+          <p>See the meeting schedule and join regular discussions with maintainers</p>
+          <i class="fas fa-arrow-right arrow"></i>
+        </a>
+
+        <a class="tile" href="https://twitter.com/strimziio" target="_blank" rel="noopener noreferrer">
+          <h6>Spread the word</h6>
+          <p>Follow us on X (formerly Twitter) and share your Strimzi story</p>
```

Comment:
We have also the YouTube channel which is mentioned in the "Spread the word" section on the current website but I am not sure how we can add it here taking into account we have just one link to jump to.
We also have LinkedIn profile as well.